_posts/2024-06-23-betadots-consulting-services.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: betadots consulting services
+date: 2024-06-23 19:17:40
+author: Tim Meusel
+tags:
+- foreman
+---
+
+[betadots GmbH](https://www.betadots.de/) joins the group of companies that
+provide professional services for Foreman!
+
+<!--more-->
+
+Well, not really right now. We already do this since years! betadots is a
+German consulting company with a focus on automation with open source
+tooling. We have a long history in the Puppet ecosystem. We assist companies
+with their planning and upgrading of their Puppet and Foreman infrastructure.
+Besides that we also offer public and inhouse trainings for Puppet and
+Foreman/Katello (with or without Puppet).
```

Comment:
How about adding the link to the public training material: https://github.com/betadots/foreman-training ?
Or is there a separate page for trainings?

Reply:
updated it